src/index.html

```diff
@@ -21,7 +24,322 @@
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="imac image"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__review">
+          <div class="stars card__review-stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review-text">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-numbers">$2,199</div>
+        </div>
+        <a
+          class="card__buy"
+          href="#Buy"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card catalog__card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="imac image"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__review">
+          <div class="stars card__review-stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review-text">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-numbers">$2,199</div>
+        </div>
+        <a
+          class="card__buy"
+          href="#Buy"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card catalog__card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="imac image"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__review">
+          <div class="stars card__review-stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review-text">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-numbers">$2,199</div>
+        </div>
+        <a
+          class="card__buy"
+          href="#Buy"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
There are multiple identical product cards. If this is intentional for layout testing, it is fine. Otherwise, consider diversifying the content to better reflect a real catalog.